app/domain/authentication/authn_jwt/identity_providers/create_identity_provider.rb

```diff
@@ -61,11 +62,24 @@ def validate_identity_configuration
         end
 
         def multiple_identities_configured
-          identity_from_decoded_token_provider.identity_available? && identity_from_url_provider.identity_available?
+          token_app_property_is_available? && identity_from_url_available?
         end
 
         def no_identities_configured
-          !identity_from_decoded_token_provider.identity_available? && !identity_from_url_provider.identity_available?
+          !token_app_property_is_available? && !identity_from_url_available?
+        end
+
+        def token_app_property_is_available?
```

Comment:
how about this?
```suggestion
        def token_app_property?
```

Reply:
But this is not really the token app property but an indication if there is such resource